google-cloud-bigtable-emulator/pom.xml

```diff
@@ -41,7 +41,7 @@
         <!-- https://github.com/googleapis/java-gcloud-maven-plugin -->
```

Comment:
please add https://github.com/googleapis/java-bigtable-hbase/blob/main/bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml#L383-L434 to this module

Reply:
sure, done

Reply:
Sure, Just pushed the skip plugin changes, thanks